eslint/babel-eslint-config-internal/index.js

```diff
@@ -1,9 +1,11 @@
+"use strict";
+
 module.exports = {
   parser: "babel-eslint",
   extends: "eslint:recommended",
   plugins: ["flowtype"],
   parserOptions: {
-    ecmaVersion: 2017,
```

Comment:
Nit: we can set [`env.es2020: true`](https://eslint.org/docs/user-guide/configuring#specifying-environments) and get rid of `parserOptions.ecmaVersion`.

Reply:
Does `babel-eslint` actually need it? :thinking:

Reply:
@nicolo-ribaudo Are you asking about `ecmaVersion`? I think it's useful to set it because rules can access it when running (example [here](https://github.com/eslint/eslint/blob/master/lib/rules/no-lone-blocks.js#L83)).

I'd be fine with setting the `es2020` env as well.

Reply:
Yeah let's do `2020`